### PR TITLE
Remove advanced options toggle

### LINKED ISSUE #1807: default don't remember pass phrase

### LINKED ISSUE #2114: show advanced options by default during setup
**So that this:**

image

**Will look more like this:**

image

### PR DESCRIPTION
**Pull request by limonte**
Closes #2114 
Closes #1807

As promised, keyboard a11y is there for free:

As well as in `add_key.html`:

### CODE REVIEW

**Comment by limonte** [comment]
Also, aligned the "Import private key" screen of `setup.htm` to be consistent in its look with `add_key.html`.

Before it looked kinda broken with different alignments in the same form:

image

After:

image

File: extension/chrome/settings/modules/add_key.htm [hunk 1]
```diff
@@ -49,27 +49,26 @@ <h1>Add Private Key <span id="spinner_container"></span></h1>
           <input id="input_passphrase" class="input_passphrase" type="text" placeholder="Private Key Passphrase" />
         </div>
         <div class="line">
-          <div class="squaredFour">
-            <input type="checkbox" value="" id="e_rememberPassphrase" name="check"
-              class="input_passphrase_save squaredFour" checked="">
-            <label class="checkboxSquaredOne" for="e_rememberPassphrase"></label>
-          </div>
-          <span class="labelCopy">Remember Pass Phrase after closing browser.</span>
+          <label>
```
Comment (tomholub) [inline extension/chrome/settings/modules/add_key.htm]: We don't seem to have any test for this particular choice.

We have a test for changing this later in settings but not during setup. I'll make a new issue for this.

**Comment by limonte** [comment]
tests are failing atm, will take care of that

**Comment by tomholub** [comment]
> tests are failing atm, will take care of that

It's because certain tests actually expect the passphrase to be present. So I guess we are testing it indirectly. For these tests that are failing, just tick this checkbox during setup.

**Comment by limonte** [comment]
> For these tests that are failing, just tick this checkbox during setup.

:+1:

**Comment by limonte** [comment]
> > tests are failing atm, will take care of that
> 
> It's because certain tests actually expect the passphrase to be present. So I guess we are testing it indirectly. For these tests that are failing, just tick this checkbox during setup.

Done :+1:

**Pull request merged**